FantasyFranchiseAPI/src/main/java/com/fafr/common/ApplicationConstants.java

```diff
@@ -21,4 +21,66 @@ public static enum PLAYER_ATTRIBUTES {
 	public enum CONFERENCES{
 		AFC,NFC;
 	}
+	
+	public enum AFC_DIVISIONS{
+		AFCEAST, AFCWEST, AFCNORTH, AFCSOUTH;
+		
+	}
+	
+	public enum NFC_DIVISIONS{
+		NFCEAST, NFCWEST, NFCNORTH, NFCSOUTH;
+		
+	}
```

Comment:
Consolidate into One Enum -- DIVISION and the values should be NORTH, SOUTH, EAST, WEST.